synapse/models/inet.py

```diff
@@ -1588,6 +1599,10 @@ def getModelDefs(self):
                         'interfaces': ('meta:taxonomy',),
                         'doc': 'A message type taxonomy.'}),
 
+                    ('inet:service:emote', ('guid', {}), {
+                        'interfaces': ('inet:service:object',),
```

Comment:
Looking at this more closely, imo inet:service:emote should inherit inet:service:action vs :object. Specifically w/:object we miss the key concept of who/what applied the reaction/emote as well as "when" the reaction was applied. We get these with :action.

Reply:
the `:creator` and `:period` properties can be used to capture this, as they have lifespan and could be subsequently removed.